scraper/mapharma.py

```diff
@@ -0,0 +1,160 @@
+import httpx
+from httpx import TimeoutException
+import json
+import logging
+import urllib.parse as urlparse
+
+from datetime import datetime, timedelta
+from pytz import timezone
+from urllib.parse import parse_qs
+from bs4 import BeautifulSoup
+
+from .pattern.scraper_request import ScraperRequest
+from .pattern.scraper_result import DRUG_STORE
+from .departements import cp_to_insee
+
+timeout = httpx.Timeout(30.0, connect=30.0)
+DEFAULT_CLIENT = httpx.Client(timeout=timeout)
+logger = logging.getLogger('scraper')
+
+insee = {}
+campagnes = {}
+
+def getName(soup):
+    name = soup.find('h3').text.strip()
```

Comment:
Ca serait être plus sûr d'être un peu plus spécifique que "le premier h3", genre plutôt : 
```suggestion
    name = soup.find(class_='pharma-block').find('h3').text.strip()
```